src/modules/initcpiocfg/main.py

```diff
@@ -21,20 +21,19 @@
 #   You should have received a copy of the GNU General Public License
 #   along with Calamares. If not, see <http://www.gnu.org/licenses/>.
 
-import libcalamares
 from libcalamares.utils import debug, target_env_call
 import os
 from collections import OrderedDict
 
 import gettext
-_ = gettext.translation("calamares-python",
+translate = gettext.translation("calamares-python",
```

Comment:
Will the string extractor still recognize the translatable strings with this rename? As far as I know, the name `_` is special in `gettext`, it is needed so that the string extraction tools find the translatable strings.

Reply:
It does not, by default. `_` and `_n` are special (er, maybe `_n` is a Calamares thing). In any case, the translations are read by `xgettext --keyword=_n:1,2 -L python` (full details, see the script `ci/txpush.sh`, which you can run with `sh ci/txpush.sh --no-tx` to just update translations locally).

You'll see that this change drops the corresponding strings from the translation catalogue.

Reply:
I think the default keywords are just `_` and `_N` (note: capital `N`!), where the latter stands for `I18N_NOOP`, i.e., mark this string for translation, but do not actually translate it here (because it is later passed to the translation function as a variable).

Reply:
Or actually, I think the NOOP is `N_`.

Reply:
You right  this needs to be reverted.